app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from os import mkdir, remove
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    original_file_name = command[1]
+
+    with open(original_file_name, "r") as file:
+        data = file.read()
+
+    path = ''
+    for folder in command[2].split('/')[:-1]:
```

Comment:
What if we don't have such a symbol (`/`)? Probably better to check for a `/` symbol in the second name